packages/dx-react-grid/src/plugins/table-column-resizing.tsx

```diff
@@ -106,9 +149,11 @@ class TableColumnResizingBase extends React.PureComponent<TableColumnResizingPro
         <Getter name="tableColumnResizingEnabled" value />
         <Getter name="tableColumns" computed={tableColumnsComputed} />
         <Getter name="tableColumns" computed={tableColumnsDraftComputed} />
+        <Getter name="nextColumnResizing" value={!!nextColumnResizing} />
```

Comment:
Why are you using the `!!` operator? According to the documentation, the `nextColumnResizing` is boolean.